pipeline-scripts/openshiftv4_on_azure.groovy

```diff
@@ -0,0 +1,150 @@
+						[$class: 'hudson.model.PasswordParameterValue', name: 'AZURE_SERVICE_PRINCIPAL_CLIENT_SECRET', value: azure_service_principal_client_secret ],
+                                                [$class: 'hudson.model.PasswordParameterValue', name: 'AZURE_BASE_DOMAIN_RESOURCE_GROUP_NAME', value: azure_base_domain_resource_group_name ],
+						[$class: 'StringParameterValue', name: 'AZURE_REGION', value: azure_region ],
+						[$class: 'StringParameterValue', name: 'OPENSHIFT_BASE_DOMAIN', value: openshift_base_domain  ],
+						[$class: 'StringParameterValue', name: 'OPENSHIFT_CLUSTER_NAME', value: openshift_cluster_name ],
+						[$class: 'StringParameterValue', name: 'OPENSHIFT_MASTER_COUNT', value: openshift_master_count ],
+						[$class: 'StringParameterValue', name: 'OPENSHIFT_WORKER_COUNT', value: openshift_worker_count ],
+						[$class: 'StringParameterValue', name: 'OPENSHIFT_MASTER_VM_SIZE', value: openshift_master_vm_size ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_WORKER_VM_SIZE', value: openshift_worker_vm_size ],
+						[$class: 'StringParameterValue', name: 'OPENSHIFT_MASTER_ROOT_VOLUME_SIZE', value: openshift_master_root_volume_size ],
+						[$class: 'StringParameterValue', name: 'OPENSHIFT_WORKER_ROOT_VOLUME_SIZE', value: openshift_worker_root_volume_size ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_CIDR', value: openshift_cidr ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_MACHINE_CIDR', value: openshift_machine_cidr ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_SERVICE_NETWORK', value: openshift_service_network ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_HOST_PREFIX', value: openshift_host_prefix ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_POST_INSTALL_POLL_ATTEMPTS', value: openshift_post_install_poll_attempts ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_TOGGLE_INFRA_NODE', value: openshift_toggle_infra_node ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_TOGGLE_WORKLOAD_NODE', value: openshift_toggle_workload_node ],
+                                                [$class: 'StringParameterValue', name: 'MACHINESET_METADATA_LABEL_PREFIX', value: machineset_metadata_label_prefix ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_INFRA_NODE_INSTANCE_TYPE', value: openshift_infra_node_instance_type ],
+                                                [$class: 'StringParameterValue', name: 'OPENSHIFT_WORKLOAD_NODE_INSTANCE_TYPE', value: openshift_workload_node_instance_type ],
```

Comment:
Lines 121 and 122 will also have to change for https://github.com/openshift-scale/scale-ci-deploy/pull/30